integration/bin/run-data-local-server.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Usage: run-data-local-server.sh
+# Builds a docker image and launches the data local service.
+
+INTEGRATION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+SRC_DIR="${INTEGRATION_DIR}/src"
+NAME=data-local
+
+echo "Building docker images for ${NAME}"
+docker build -t ${NAME} ${SRC_DIR}
+
+docker create \
+       -i \
+       -t \
+       --rm \
+       -p 5000:5000 \
```

Comment:
Would it be worth commenting why we are forwarding port 5000, or is it obvious to those with more docker-fu than I?